Query was executed successfully

[(1,2,1),(1,3,2.1),(1,4,2.2),(2,1,3.3),(2,3,4),(2,4,5),(3,1,7.1),(3,2,8.2),(3,3,9.3),(4,3,10.4),(4,4,11),(4,5,12.1),(5,1,13.2),(5,2,14.3),(5,3,15.4),(5,4,16),(7,2,20.4),(7,3,21),(7,4,22),(7,5,23.1),(7,6,24.2),(8,7,25.3),(8,1,26.4),(8,2,27.5),(8,3,28),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),(),()]

SCIDB QUERY : <CREATE ARRAY Data < J : int64 NULL DEFAULT 9999 > [ I(int64)=*,10,0, V(double)=*,10,0 ]>
Query was executed successfully

SCIDB QUERY : <redimension_store(Raw_Test_Data, Data)>
[An error expected at this place for the query "redimension_store(Raw_Test_Data, Data)". And it failed with error code = scidb::SCIDB_SE_INFER_SCHEMA::SCIDB_LE_OP_REDIMENSION_STORE_ERROR5. Expected error code = scidb::SCIDB_SE_INFER_SCHEMA::SCIDB_LE_OP_REDIMENSION_STORE_ERROR5.]

SCIDB QUERY : <scan (Raw_Test_Data)>
[(1,2,1),(1,3,2.1),(1,4,2.2),(2,1,3.3),(2,3,4),(2,4,5),(3,1,7.1),(3,2,8.2),(3,3,9.3),(4,3,10.4),(4,4,11),(4,5,12.1),(5,1,13.2),(5,2,14.3),(5,3,15.4),(5,4,16),(7,2,20.4),(7,3,21),(7,4,22),(7,5,23.1),(7,6,24.2),(8,7,25.3),(8,1,26.4),(8,2,27.5),(8,3,28)]

SCIDB QUERY : <scan (Data)>
[[]]

Query was executed successfully

Query was executed successfully

